src/tokenizer/stop_word_filter/stopwords.rs

```diff
@@ -0,0 +1,2115 @@
+/*
+Copyright (c) 2001, Dr Martin Porter
```

Comment:
Can you add a comment/url to explain the origin of this data?

Reply:
Done.